packages/quick_actions/quick_actions/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for creating shortcuts on home screen, also known as
   Quick Actions on iOS and App Shortcuts on Android.
 repository: https://github.com/flutter/plugins/tree/main/packages/quick_actions/quick_actions
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+quick_actions%22
-version: 0.6.0+10
+version: 0.7.0
```

Comment:
The way Dart versioning works for pre-1.0, this is a breaking version change (see link in the PR checklist about Dart versions). Is there anything about this change that actually breaks clients?